frontend/src/components/app/app.tsx

```diff
@@ -2,14 +2,19 @@ import { AppRoute } from 'common/enums/enums';
 import { FC } from 'common/types/types';
 import { Routes, Route, AuthorizedWrapper } from 'components/common/common';
 import { Auth } from 'components/auth/auth';
+import { AuthedMenu } from 'components/common/common';
```

Comment:
You already have some components imported from `components/common/common` so move AuthedMenu import there (+ I think `Auth` component also could be imported from `components/common/common`)